intellij/.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# Dependabot configuration:
```

Comment:
This time should be deleted and some of its content should be added to the main dependabot configuration